pkg/webhooks/regularuser/common/regularuser.go

```diff
@@ -287,7 +288,7 @@ func (s *RegularuserWebhook) authorized(request admissionctl.Request) admissionc
 
 // isMustGatherAuthorized check if request is authorized for MustGather CR
 func isMustGatherAuthorized(request admissionctl.Request) bool {
-	return slices.Contains(request.UserInfo.Groups, ceeGroup)
```

Comment:
I don't think ceeGroup should be removed here. We want CEE to be able to use must-gather-operator

Reply:
I just added the ticket to this PR - there should be more details there.